packages/fxa-auth-server/scripts/stripe-products-and-plans-to-firestore-documents/stripe-products-and-plans-converter.ts

```diff
@@ -26,16 +26,43 @@ import {
   PLAN_EN_LANG_ERROR,
   getLanguageTagFromPlanMetadata,
 } from './plan-language-tags-guesser';
+import { promises as fsPromises, constants } from 'fs';
+import path from 'path';
 
 const DEFAULT_LOCALE = 'en';
 
+export enum OutputTarget {
+  Local = 'local',
+  Firestore = 'firestore',
+}
+
 function isGoogleTranslationApiError(err: any) {
   return (
     err.code >= 400 &&
     err.response?.request?.href?.includes('translation.googleapis')
   );
 }
 
+async function createDir(path: string) {
+  try {
+    // Check if the folder exists.
+    // https://nodejs.dev/learn/working-with-folders-in-nodejs#check-if-a-folder-exists
+    await fsPromises.access(path, constants.W_OK);
+  } catch (err) {
+    // If folder doesn't exist, access throws error. So create folder.
+    await fsPromises.mkdir(path, { recursive: true });
```

Comment:
I don't think we should over-rotate on this for a script, but it is possible that the `err` is due to that the dir exists but not writable.  I'm not asking for a change since presumably whoever run this script is aware of file permissions and hopefully can parse nodejs errors.